### DISCUSSION TITLE
What about Pycharm users ?

### DISCUSSION THREAD

**Discussion by remyAuneor** [odoo/odoo-ls#80] [Ideas]
Is a Pycharm extension planned ? :)

**Comment by fda-odoo** [comment]
Hello :)
As PyCharm now supports LSP since 3 months, we plan to create an extension for it. The development didn't start already, as we prefer not have too much IDE at first. Moreover, PyCharm decided to restrict this functionnality to Premium users, and we want to see if it will change or not too.

**Reply by remyAuneor** [reply]
Owh I wasn't aware of that premium limitation ! Thanks for the answer !
